test/jdk/sun/security/pkcs12/KeytoolOpensslInteropTest.java

```diff
@@ -447,7 +598,28 @@ private static void check(
         Asserts.assertEQ(expectedKey, actualKey, label + "-key");
     }
 
-    static OutputAnalyzer keytool(String s) throws Throwable {
+    private static OutputAnalyzer keytool(String s) throws Throwable {
         return SecurityTools.keytool(s);
     }
-}
+
+    private static String getSystemOpensslPath() {
+        if(verifyOpensslVerion("/usr/bin/openssl")) {
+            return "/usr/bin/openssl";
+        } else if(verifyOpensslVerion("/usr/local/bin/openssl")) {
+            return "/usr/local/bin/openssl";
+        }
+        return null;
+    }
+
+    private static boolean verifyOpensslVerion(String path) {
+        try {
+            ProcessTools.executeCommand(path, "version")
+                    .shouldHaveExitValue(0)
+                    .shouldMatch("1.1.*");
```

Comment:
Precisely the regex should be `1\\.1\\..*`, or maybe you can simply call `.shouldContain("1.1.")`.

Reply:
Changed it in to .shouldContain("1.1.")